src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/EdgeCasesTests.cs

```diff
@@ -105,6 +105,7 @@ public static void ZeroLengthContent_RoundTrip()
                 }
                 catch (CryptographicException) when (PlatformDetection.IsFullFramework) // Expected on full FX
                 {
+                    return;
```

Comment:
BTW, I think it's preferable to [SkipOnTarget] for this one. This test case was isolated into its own Fact precisely because of the change in behavior from desktop. In fact, if there's a naming convention already for tests that exercise "not net46" behavior, I think it'd be good to rename the test accordingly. I've seen tests suffixed with "_net46" for other full-framework-specific tests.